frontend/src/features/proposals/ProposalsList.tsx

```diff
@@ -23,7 +23,7 @@ const formatRunCycle = (date: string) => {
   return `${year} - ${period}`
 }
 
-const ExpandedCell = memo(({ Component, isExpanded }) => {
+const ExpandedCell = memo(function ExpandedCell({ Component, isExpanded }) {
```

Comment:
I don't think that in this case it does make a difference and if you are aware of this and already considered it just ignore this comment. Anonymous functions and named one are almost 100% equivalent but have subtle differences (and once I got bit by it :zombie: ). Below is a screenshot from MDN (taken from [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions)) where these are explained: 

![image](https://github.com/user-attachments/assets/f38d34d9-54a5-4665-811a-2361a32fc208)

Reply:
The previous implementation fails the linting as we're passing an anonymous function (e.g., an arrow function) inside `React.memo`. One has to use a named function instead, which I have simply added as such inline.